src/talos/example_config.toml

```diff
@@ -16,9 +16,24 @@ forced_panels = [1, 2, 3]
 # we treat the gene as new. New/Recent genes are highlighted in the report with a Star next to the symbol
 within_x_months = 6
 
-[FindGeneSymbolMap]
-# much higher than this and we get into throttling issues
-chunk_size = 800
+[GeneratePanelData.manual_overrides]
+# this section permits the manual addition of genes to the panel data
+# each gene here will be folded into the panelapp data after standard API queries have taken place
+# if the gene was already identified for this analysis, the MOI provided here will take precedent
+# if the gene here was not identified through panelapp, a new entity will be added, and the panel ID
+# associated with it will be the number zero (panel IDs must all be numeric, 0 is not in use)
+[[GeneratePanelData.manual_overrides.genes]]
```

Comment:
This double-bracketed syntax means we can add an unlimited number of these sections to the config file as a list. These can be written manually, or done by writing a list of dictionaries and using `print(toml.dumps(THE_LIST))` on the object.